docs/_includes/Statements.md

```diff
@@ -1,6 +1,6 @@
 # Statements
 ````
-Stmt = ( BlockStmt | AssertStmt | AssumeStmt | PrintStmt | UpdateStmt
+Stmt = ( BlockStmt | AssertStmt | AssumeStmt | ExpectStmt | PrintStmt | UpdateStmt
```

Comment:
An `expect` statement can also be used in a `StmtExpr`.

Reply:
Added